src/website/views/components/label/_examples.pug

```diff
@@ -0,0 +1,178 @@
+h2 Examples
+h3 Base
+.example.-mb--3
+  .-p--3
+    chi-label Label
+  .example-tabs.-pl--2
+    ul.chi-tabs#example-base
+      li.-active
+        a(href='#webcomponent-base') Web Component
+      li
+        a(href='#html-base') HTML Blueprint
+  #webcomponent-base.chi-tabs-panel.-active
+    :code(lang="html")
+      <chi-label>Label</chi-label>
+  #html-base.chi-tabs-panel
+    :code(lang="html")
+      <label class="chi-label">Label</label>
+
+h3 Required
+.-text
+  | Add attribute <code>required</code> to indicate required fields.
```

Comment:
Should we change `text-decoration: underline dotted;` ?

Reply:
@janibolkvadze Yes, can we move the asterisk to the left of the label, set margin-left to 0, and set margin-right to 0.25rem? Left placement may scale better as it will always be a consistent position despite the labels character length and it won't conflict with info icons placed to the right of the label.